src/libreminesgameengine.h

```diff
@@ -39,7 +39,7 @@ class LibreMinesGameEngine: public QObject
 
         CELL_STATE state;
         bool isHidden;
-        bool hasFlag;
+        int FlagType;
```

Comment:
The name of the attribute `FlagType` should be `flagType`.

I think it is appropriate to create an enum for the Flag Type. Like so:

```cpp
    enum class FlagType : quint8
    {
        NoFlag = 0,
        HasFlag = 1,
        Question = 2
    };
```

That way, all references to FlagType would use this enum. Like:
```cpp
...
		if (cellGE.flagType == FlagType::Question)
			img = fieldTheme.getPixmapQuestion().toImage();
...
```